### ISSUE TITLE
Quarto details

### ISSUE THREAD

**Issue by jthomasmock** [r-lib/sessioninfo#75]
Howdy! 

Would it be possible to return the `quarto` version as part of session info?

`pandoc` is currently reported, but would be great to also have the `quarto` version itself at the bottom of the Session Info section.

```
─ Session info ─────────────────────────
 setting  value
 version  R version 4.1.1 (2021-08-10)
 os       macOS Monterey 12.2.1
 system   aarch64, darwin20
 ui       RStudio
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/Chicago
 date     2022-04-18
 rstudio  2022.06.0-daily+217 Spotted Wakerobin (desktop)
 pandoc   2.17.1.1 @ /Applications/RStudio.app/Contents/MacOS/quarto/bin/ (via markdown)
```

A quick attempt at implementation based off the `get_pandoc_version()` function.

``` r
get_quarto_version <- function() {
  if (isNamespaceLoaded("quarto")) {
    path <- quarto::quarto_path()
    ver <- system("quarto -V", intern = TRUE)
    if (is.null(path)) {
      "NA (via quarto)"
    } else {
      paste0(ver, " @ ", path, "/ (via quarto)")
    }
  } else {
    path <- Sys.which("quarto")
    if (path == "") {
      "NA"
    } else {
      ver <- system("quarto -V", intern = TRUE)
      paste0(ver, " @ ", path)
    }
  }
}


get_quarto_version()
#> [1] "0.9.172 @ /usr/local/bin/quarto"
```

<sup>Created on 2022-04-18 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by gaborcsardi** [comment]
Good idea, thanks!

**Comment by cderv** [comment]
If that helps, we could add a `quarto::quarto_version()` or else in the R package.

**Comment by jthomasmock** [comment]
> If that helps, we could add a `quarto::quarto_version()` or else in the R package but there need a way to retrieve the info if the package is not installed anyway

Probably makes sense to expose the version via quarto R package itself but agreed that there needs to be an option ignoring the quarto R package being available.

**Comment by jthomasmock** [comment]
Bumping this now that Quarto is fully public and there will be a 1st stable release (`1.0.36`) and then subsequent stable releases.

Happy to turn into a PR if that's ok!

**Comment by jennybc** [comment]
I just came here to open this issue, but instead I will make this supportive comment :grinning:

Semi-related issue I opened over in the quarto-actions repo: https://github.com/quarto-dev/quarto-actions/issues/52

**Comment by jthomasmock** [comment]
Howdy! I wanted to followup here - would you be open to a PR for this feature?

**Comment by gaborcsardi** [comment]
Fixed in #87.
